plotly/tools.py

```diff
@@ -2353,332 +2219,477 @@ def create_ohlc(open, high, low, close,
                  datetime(year=2014, month=2, day=10)]
 
         # Create ohlc
-        fig = FF.create_ohlc(open_data, high_data,
+        fig = FF.create_candlestick(open_data, high_data,
             low_data, close_data, dates=dates)
 
-        py.iplot(fig, filename='finance/simple-ohlc', validate=False)
+        py.iplot(fig, filename='finance/simple-candlestick', validate=False)
         ```
         """
         if dates is not None:
-            TraceFactory.validate_equal_length(open, high, low, close, dates)
+            FigureFactory.validate_equal_length(open, high, low, close, dates)
         else:
-            TraceFactory.validate_equal_length(open, high, low, close)
+            FigureFactory.validate_equal_length(open, high, low, close)
         FigureFactory.validate_ohlc(open, high, low, close, direction,
                                     **kwargs)
 
         if direction is 'increasing':
-            ohlc_incr = FigureFactory._make_increasing_ohlc(open, high,
-                                                            low, close,
-                                                            dates, **kwargs)
-            data = [ohlc_incr]
+            candle_incr_data = FigureFactory._make_increasing_candle(open,
+                                                                     high,
+                                                                     low,
+                                                                     close,
+                                                                     dates,
+                                                                     **kwargs)
```

Comment:
:cow2: there's no *hard* rule on the hanging format, we just prefer it in general. perhaps?

```
candle_incr_data = FigureFactory._make_increasing_candle(
    open, high, low, close, dates, **kwargs
)
```